examples/python/assistant.ipynb

```diff
@@ -0,0 +1,150 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "569d2618-ff32-466a-8bec-eeb967ee364b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import onnxruntime_genai as og\n",
+    "import argparse\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "add45ace-14be-4ab3-a68c-303aebeea18c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loading model...\n",
+      "Model loaded in 41.10 seconds\n"
```

Comment:
Loading phi2 on my machine is much faster than this.
Since this is really machine dependent, can we avoid having the output cached in the notebook?

Reply:
We can. How do we do that?

Reply:
I added a suggestion. Just remove the text block from the ipynb

Reply:
Do you mean manually, before checking it in?

Reply:
Yes, or you could also click on the clear cell outputs before checking it in.